packages/api/cms-api/src/mailer/mailer.service.ts

```diff
@@ -30,13 +36,31 @@ export class MailerService {
      * Sends a mail and logs it in the database.
      * If mailerConfig.sendAllMailsTo is set, the mail will be sent to this address instead of the `to` address and `bcc` will be omitted. This can
      * be used to prevent non-prod environments from sending mails to real customers.
-     * @param type Mail type, e.g. order confirmation, order cancellation, etc. to filter in the mailer log
+     * @param mailTypeForLogging Mail type, e.g. order confirmation, order cancellation, etc. to filter in the mailer log
```

Comment:
This is an incompatible change! If 8.0 is released without this PR that is a problem. (Theoretical only, but you really have to be careful when developing a library - in this case we even already discussed the naming of this option in the original PR)